src/components/SQForm/SQFormTextarea.tsx

```diff
@@ -1,12 +1,39 @@
 import React from 'react';
-import PropTypes from 'prop-types';
 import TextField from '@material-ui/core/TextField';
 import Grid from '@material-ui/core/Grid';
-import {useFormikContext} from 'formik';
-
+import {useFormikContext, FormikProps} from 'formik';
 import {useForm} from './useForm';
+import {TextareaAutosizeProps, TextFieldProps} from '@material-ui/core'
+import BaseFieldProps from '../../types/BaseFieldProps';
 import {toKebabCase} from '../../utils';
 
+interface SQFormTextareaProps extends BaseFieldProps {
+  /** Placeholder text used inside the input field to provide hints to the user */
+  placeholder: string;
+  /** Disabled property to disable the input if true */
+  isDisabled: boolean;
+  /** Required property used to highlight input and label if not fulfilled */
+  isRequired: boolean;
+  /** Custom onBlur event callback */
+  onBlur: React.FocusEventHandler;
```

Comment:
To be more explicit:
```suggestion
  onBlur: React.FocusEventHandler<HTMLTextAreaElement>;
```